Pipfile.lock

```diff
@@ -0,0 +1,68 @@
+{
+    "_meta": {
+        "hash": {
+            "sha256": "92c271cf58b7b905ee81b93ca29f1fdf58b0b620f09c22a64d17686c6078fb21"
+        },
+        "pipfile-spec": 6,
+        "requires": {
+            "python_version": "2.7"
```

Comment:
Same note on Python 3, since we are looking towards the Python 2 EoL soon anyways.